CHANGES/8845.bugfix.rst

```diff
@@ -1 +1 @@
-Changed behaviour when returning an invalid response to send a 500 response -- by :user:`Dreamsorcerer`.
+Changed behavior when returning an invalid response to send a 500 response -- by :user:`Dreamsorcerer`.
```

Comment:
This showed up as a spelling, but not sure if we should be using American English or British English?

Reply:
I tried asking about the spelling in other projects like pip and learned that they don't enforce a specific variant, accepting anything people write. And so I thought we could default to that here too.

I think Sam uses the British variant because of where he's based and that's okay.

On a related note, I've been meaning to explore integrating https://vale.sh to support style consistency efforts. I assume the tool would force us to select one way or the other, whenever we get to it. For now, I would only fix things that the linters enforce but wouldn't be too vocal about non-enforced personal preferences in reviews.

To summarize, I don't have a strong opinion here so long as others don't either.